crates/core/src/install.rs

```diff
@@ -211,8 +212,7 @@ pub async fn install_dependencies(
             let d = dep.clone();
             #[cfg(feature = "cli")]
             let p = progress.clone();
-
-            let lock = locks.iter().find(|l| l.name() == dep.name()).cloned();
+            let lock = locks.par_iter().find_any(|l| l.name() == dep.name()).cloned();
```

Comment:
This is slower than normal find 
![image](https://github.com/user-attachments/assets/5b2bdbb2-5201-48cf-876e-cee18b5dd88d)